product_warranty/models/product.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    warranty_time = fields.Integer(string='Warranty Time',
```

Comment:
Could it be that the product variants have different warranty times @BT-mfasola ? I guess so. Then this should be in product.product

Reply:
Good point. modified.